### PR TITLE
Now we can run on now

### PR DESCRIPTION
**Pull request by deonjo**
https://arrangespace-server.gideon.now.sh/health

### CODE REVIEW

**Review by dwosk — CHANGES_REQUESTED** [review]
Awesome! I'll take a look. Can you also make sure the lint passes? 

`npm run lint`

File: index.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
+'use strict'
 const helmet = require('helmet');
 const express = require('express');
-const app = module.exports = express();
+const app = express();
+module.exports = app
```
Comment (dwosk) [inline index.js]: Does it still work if we move this to the bottom of index.js? I think it's more node idiomatic that way.

Comment (deonjo) [reply index.js]: it doesn't work if it's below 
let db = require('./db/database_mongo');

**Comment by deonjo** [comment]
Fixed ESlint issues

**Review by dwosk — APPROVED** [review]

**Pull request merged**